websites/F/fbox.to/metadata.json

```diff
@@ -18,9 +18,9 @@
 	"description": {
 		"en": "fbox online is free movies streaming website with zero ads, it allows you watch free movies online, watch free tv series online in high quality for free"
 	},
-	"url": "fbox.to",
+	"url": "fboxz.to",
```

Comment:
no point bcuz of the regex?

Reply:
Regex doesn't take into account the trailing `z` in the domain